SConstruct

```diff
@@ -4,6 +4,16 @@ import tarfile
 import zipfile
 from SCons.Script.SConscript import SConsEnvironment
 
+def check_output(s, **kwargs):
+    p = subprocess.Popen(s, stdout=subprocess.PIPE, **kwargs)
+    (outdata, errdata) = p.communicate()
+    r = p.returncode
+    assert r is not None
+    if r != 0:
+        raise subprocess.CalledProcessError(r, s)
+    return outdata
+subprocess.check_output = check_output
```

Comment:
This will *always* overwrite the `subprocess.check_output` function with the provided substitute. This should only take effect if the imported subprocess module does *not* already have a check_output function.